core/google/cloud/environment_vars.py

```diff
@@ -33,9 +30,6 @@
 BIGTABLE_EMULATOR = 'BIGTABLE_EMULATOR_HOST'
 """Environment variable defining host for Bigtable emulator."""
 
-CREDENTIALS = 'GOOGLE_APPLICATION_CREDENTIALS'
```

Comment:
It'll still be useful to have this around for giving nice error messages when system tests fail. (Also this indicates to me that you just maybe haven't run system tests, is this the case?)